crates/torii/graphql/src/query/mod.rs

```diff
@@ -93,7 +92,10 @@ fn parse_nested_type(
             }
         })
         .collect();
-
+    println!("target_id: {:?}", target_id);
+    println!("target_type: {:?}", target_type);
+    println!("nested_mapping: {:?}", nested_mapping);
+    let target_type = format!("{}_{}", target_id, target_type);
```

Comment:
Maybe call this `namespaced` for more context. These recursive calls are a pain to work with...